static/js/components/NotificationContainer_test.js

```diff
@@ -91,4 +91,50 @@ describe("NotificationContainer component", () => {
       hiddenNotifications: new Set(["message1"])
     })
   })
+
+  it("shows cms notifiction", async () => {
+    const { inner } = await render({
+      ui: {
+        userNotifications: {
+          "cms-site-wide-notification": {
+            type:  ALERT_TYPE_TEXT,
+            props: { text: "Cms Notifictaion", persistedId: 1 }
```

Comment:
Nit:
```suggestion
            props: { text: "Cms Notification", persistedId: 1 }
```
This spelling occurs many times below